Setup.Aggregator/T2018.xslt

```diff
@@ -18,14 +18,14 @@
   <!--Give Component ID prefix -->
   <xsl:template match="wix:Component/@Id">
     <xsl:attribute name="{name()}">
-      <xsl:value-of select="concat('F2018_', .)" />
```

Comment:
I think this is the mistake: you need to rollback the changes to Setup.Aggregator/T2018.xslt